### ISSUE TITLE
YAM crashes every now and then

### ISSUE THREAD

**Issue by rmsyam** [jens-maus/yam#628]
Under MorphOS YAM crashes every now and then, can't trace why though, suddenly the system is blocked while working with YAM. Log's attached.

**Comment by jens-maus** [comment]
You haven't provided any crashlog or more information so far. So please provide that so that we can investigate the issue adequately.

**Comment by rmsyam** [comment]
I wasn't able to add the log's :-( It want let me, sorry! Can I send the log's in another way?

**Comment by jens-maus** [comment]
Why it shouldn't let you add the log? All you have to do it to zip it as github tella you for unknoqn file formats

**Comment by rmsyam** [comment]
Here they are...

**Comment by jens-maus** [comment]
Sorry, but there is nothing. Please make sure to use the web interface of github with a modern browser and upload the logfile zip archived using drag&drop.

**Comment by rmsyam** [comment]
Well what is a modern webbrowser? I'm using YAM under MorphOS and the only webbrowser available is OWB! Unfortunately it seems not to work, neither drag&drop neither selecting the files, neither zip compressed, neither plain txt files. Hmmm...

**Comment by jens-maus** [comment]
Well, then obviously OWB is not as modern as it claims to be ;) But in fact OWB on AmigaOS4 shows the exact same behavior here (haven't tested the latest OWB-MUI version, thought). What a pity that Amiga is always years behind what the rest of the world actually is using, but well. 

So the only possibility is now that you either inline the log output between threes apostrophe characters (`) or you simply send us the logfile via private mail and we will upload it using firefox on a modern OS. Sorry for all the fuzz, but I think you should stress the OWB developers now to actually update their web-engine to be actually compatible to the latest HTML5 standard.

**Comment by jens-maus** [comment]
[log_09-05-16_yam_crash.zip](https://github.com/jens-maus/yam/files/313389/log_09-05-16_yam_crash.zip)
[log_10-05-16_yam_crash.zip](https://github.com/jens-maus/yam/files/313390/log_10-05-16_yam_crash.zip)

**Comment by tboeckel** [comment]
Just for the records, even an 100% up to date browser is no guarantee to be able to upload attachments. Jens, as you know I was not able to upload any file from Windows 7 running the latest Firefox, while it was possible running the same Firefox version on Linux. I yet have to test Chrome more intensively. But I think this is enough proof that there is something wrong on github's side.

**Comment by jens-maus** [comment]
@tboeckel You are right, but I also never fully understand why it wouldn't work with firefox on windows 7. I will see if there are already reports somewhere on github that verify your observations.

**Comment by jens-maus** [comment]
@rmsyam I just had a quick look at your provided crash logs. To me this looks more like general memory trashing causing your system to go wild and as a consequence crashing while YAM is doing its jobs. None of the stack traces in your crash logs directly link to YAM functions that immediately crash. So the question is if your problem is actually reproducible and if so please download the debug version of YAM and execute it so that additional debug information is recorded.

**Comment by rmsyam** [comment]
ok, I will use the debug version of YAM and send you the log asap. On the other hand I will submit your report to the MorphOS developpers is well, so they can have a look also.
Question: does the debug version of YAM need to be started in a special way or can I just run it?

**Comment by tboeckel** [comment]
Just run it. If we need special flags to be set in ENV:yamdebug we will tell you.
